<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDialog</class>
 <widget class="QDialog" name="MainWindowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowDialog</string>
  </property>
  <widget class="QPushButton" name="openSecondWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Second Window</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LatLon">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>32.12345678,-99.98765432</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="startStopgroupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="startMotion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="stopMotion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="dsHeading">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="value">
    <double>45.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="mphLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MPH</string>
   </property>
  </widget>
  <widget class="QLabel" name="mphLabel_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Degrees</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsSpeed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
   <property name="value">
    <double>106.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Heading</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="startStopgroupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="metricButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Metric</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="imperialButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imp</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="dsAlt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
   <property name="value">
    <double>250.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="altLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FEET</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="mSecond">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>160</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="value">
    <double>-2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="fsLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>F/S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vspd</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
